cypress/e2e/brewery/DiscardAndContinue.cy.js

```diff
@@ -120,7 +120,7 @@ describe('Discard and continue go to workspaces', () => {
 
     Scenarios.getScenarioViewTab(60).should('be.visible');
 
-    Workspaces.getHomeButton().should('be.visible').click();
+    Workspaces.switchToWorkspaceView();
```

Comment:
This makes tests much easier to read, thank you for this generic function :smile: